tests/max_config/nidaqmxMaxConfig.ini

```diff
@@ -81,17 +79,15 @@ SampClk.Rate = 1000
 SampTimingType = Sample Clock
 SampClk.Src = 
 Author = Test Author
-AllowInteractiveEditing = True
 
 [DAQmxScale double_gain_scale]
+Descr = Twice the gain
 Lin.Slope = 2
 Lin.YIntercept = 0
 PreScaledUnits = Volts
 ScaledUnits = 
 ScaleType = Linear
 Author = Test Author
-AllowInteractiveEditing = True
```

Comment:
Did you remove AllowInteractiveEditing on purpose?

Reply:
Nope.. not sure why MAX would have done that. Grr...